MovieApp/PresentationLayer/Components/NavigationController/BaseNavigationController.swift

```diff
@@ -0,0 +1,9 @@
+import UIKit
+
+class BaseNavigationController: UINavigationController {
```

Comment:
Where is this used?

Reply:
`SceneDelegate` creates a `BaseNavigationController` instance and passes it to the `MainRouter` during initialization
```
        let window = UIWindow(windowScene: windowScene)
        self.window = window
        
        router = MainRouter(navigationController: BaseNavigationController())
        router?.start(in: window)
```

Reply:
OK but why can't you use just UINavigationController?

Reply:
You can delete this class.

Reply:
I subclassed `UINavigationController` because according to a few resources online that is the recommended way to set the status bar style. Either that or writing an extension.
Is there another way to achieve this?

Reply:
We use extensions.